common/bitutil/compress.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2013 The Go Authors. All rights reserved.
```

Comment:
I'll change the copyright notice. It should be us, not upstream Go. Will add some tests and push a second commit on top that fixes this too :)